server/metrics/metrics.go

```diff
@@ -121,7 +138,7 @@ func NewMetrics(info InstanceInfo) Metrics {
 	m.pluginStartTime = prometheus.NewGauge(prometheus.GaugeOpts{
 		Namespace:   MetricsNamespace,
 		Subsystem:   MetricsSubsystemSystem,
-		Name:        "plugin_start_time",
+		Name:        "plugin_start_timestamp_seconds",
```

Comment:
These, among the rest of the changes, are all intended to align with https://prometheus.io/docs/practices/naming/.